js/base/functions/throttle.js

```diff
@@ -15,61 +15,67 @@ class Throttle {
             maxCapacity: 2000,
             tokens: 0,
             cost: 1.0,
-        }
-        Object.assign (this.config, config)
-        this.queue = []
-        this.running = false
+        };
+        Object.assign (this.config, config);
+        this.queue = [];  // requests to be sent
+        this.running = false;
+    }
+
+    async resolve (resolver) {
+        resolver (); // method for api endpoint
```

Comment:
so this would be the big change that would get the available limit to fill up faster. Would it work how I think?